src/ophys_etl/transforms/roi_transforms.py

```diff
@@ -127,3 +129,149 @@ def crop_roi_mask(roi_mask: coo_matrix) -> coo_matrix:
+    """
+    This functions transforms ROI mask data from COO format
+    to the old format used in older segmentation. This function
+    writes only the data associated with the mask.
+    Parameters
+    ----------
+    coo_mask: coo_matrix
+        The coo roi matrix to be converted to old segmentation mask format
+
+    Returns
+    -------
+    Dict:
+        A dictionary that contains the mask data from the coo matrix in the
+        old segmentation format
+        {
+            'x': int (x location of upper left corner of roi in pixels)
+            'y': int (y location of upper left corner of roi in pixels)
+            'width': int (width of the roi mask in pixels)
+            'height': int (height of the roi mask in pixels)
+            'mask_matrix': List[List[bool]] (dense matrix of roi mask)
+        }
+    """
+    bounds = roi_bounds(coo_mask)
+    height = bounds[1] - bounds[0]
+    width = bounds[3] - bounds[2]
+    mask_matrix = crop_roi_mask(coo_mask).toarray()
+    mask_matrix = np.array(mask_matrix, dtype=bool)
+    old_roi = {
+        'x': int(bounds[0]),
+        'y': int(bounds[2]),
+        'width': int(width),
+        'height': int(height),
+        'mask_matrix': mask_matrix.tolist()
+    }
+    return old_roi
+
+
+def _check_exclusion(old_roi: Dict,
+                     movie_shape: Tuple[int, int]) -> Tuple[List[int], bool]:
+    """
+    Checks if roi in old styling needs to be excluded as it breaks
+    one of the defined conditions within this function. Returns a list of
+    ints corresponding to the codes for exclusion. Also return a boolean
+    indicating if the roi is valid or not.
+    Parameters
+    ----------
+    old_roi: Dict
+        An ROI stored in old dictionary format that minimally contains the
+        following values.
+        {
+            'x': int (x location of upper left corner of roi in pixels)
+            'y': int (y location of upper left corner of roi in pixels)
+            'width': int (width of the roi mask in pixels)
+            'height': int (height of the roi mask in pixels)
+            'max_correction_up': int
+            'max_correction_down': int
+            'max_correction_left': int
+            'max_correction_right': int
+        }
+
+    movie_shape: Tuple[int, int]
+        The frame shape of the movie from which ROIs were extracted in order
+        of: (height, width).
+    Returns
+    -------
+    Tuple[List[int]], bool]
+        The tuple contains a list of exclusion codes as it 0 index and
+        a bool indicating if the roi is valid as its 1 index value
+    """
+    valid_roi = True
+    exclusion_labels = []
+
+    # check if roi exists partly or wholey outside motion border
+    movie_height, movie_width = movie_shape[0], movie_shape[1]
+    furthest_right_pixel = old_roi['x'] + old_roi['width']
+    furthest_down_pixel = old_roi['y'] + old_roi['height']
+    if (old_roi['x'] <= old_roi['max_correction_left'] or
```

Comment:
Can you double check if the old pipeline uses `>`/`<` or `>=`/`<=` for this motion border exclusion definition?

Reply:
It is >= and <=